src/core/evaluator.js

```diff
@@ -1122,6 +1122,12 @@ class PartialEvaluator {
         case "Type":
           break;
         case "LW":
+          if (typeof value !== "number") {
+            warn("Line width is not a number");
```

Comment:
Please update this one a little bit, to have a more consistent warning format.
```suggestion
            warn(`Invalid LW (line width): ${value}`);
```